HW_1/task1.py

```diff
@@ -0,0 +1,15 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+num = int(input('Введите трезначное число: '))
+if num > 99 and num < 1000:
+    n1 = num // 100
+    n2 = (num // 10) % 10
```

Comment:
У операций деления одинаковый приоритет.
```suggestion
    n2 = num // 10 % 10
```